qml/pages/MessageListView.qml

```diff
@@ -159,9 +203,10 @@ SilicaListView {
     }
 
     function markLatest() {
-        if (latestRead != "") {
-            slackClient.markChannel(channel.id, latestRead)
-            latestRead = ""
+        if (furthestRead != "") {
+            slackClient.markChannel(channel.id, furthestRead)
```

Comment:
`setLastRead` already calls `slackClient.markChannel()` so this call here is redundant (we actually call the API twice now)

Reply:
Oh, you're right!